### ISSUE TITLE
[Bug] Unable to declare war on other factions.

### ISSUE THREAD

**Issue by AceofSpades303** [ToberoCat/ImprovedFactions_new#76]
**Describe the bug**
Can't declare war on other factions. 

**Error log from console**
Error: NoLocalizationFound

**To Reproduce**
1. /f relation
2. /f relation war
3. /f relation war <faction name>

**Expected behavior**
The command works find for alliance, but for war it doesn't seem to work. I expected to broadcast a successful declaration of war to the server, but all I got was the error

**Screenshots**
![Base Profile Screenshot 2023 06 21 - 13 41 16 44](https://github.com/ToberoCat/ImprovedFactions_new/assets/137324551/eeec169c-f5b1-432e-9e9a-acbb7a01cf66)

![Base Profile Screenshot 2023 06 21 - 13 42 06 09](https://github.com/ToberoCat/ImprovedFactions_new/assets/137324551/d352d346-862a-4d34-b484-539b7d410a03)

![Base Profile Screenshot 2023 06 21 - 13 42 22 19](https://github.com/ToberoCat/ImprovedFactions_new/assets/137324551/d544698f-f9ec-4a99-a008-753206a50453)

**Version (please complete the following information):**
 - Plugin: 1.6.2
 - Server: 1.19.2

**Additional context**
I've been having a few other issues with "Localization" in some menus and others.

**Comment by AceofSpades303** [comment]
Hi, thank you, I noticed when I ran the /f who command my faction was in a war. However, when we actually fought, we could not break each other’s blocks on claimed territory. Is this a feature? In order to have our fight to the fullest I had to disable the plugin. If this is a feature, is there a way to disable this?

**Comment by ToberoCat** [comment]
No, sorry. Such a feature isn't in place. Relations in general don't have much of an impact on the gameplay with this plugin
